From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 9 Jan 2025 23:27:13 -0600
Subject: [PATCH] downstream: Suppress runpath in openmp

The libraries for openmp are installed into libdir, so RUNPATH is unnecessary. If this argument is not set then a relative RUNPATH is added to `llvm-omp-*` binaries which Boulder can't currently handle.

Origin: vendor
Forwarded: not-needed
Last-Update: 2026-08-28
---
 offload/tools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/offload/tools/CMakeLists.txt b/offload/tools/CMakeLists.txt
index 7c35cd4c80b9..6f5db4534b22 100644
--- a/offload/tools/CMakeLists.txt
+++ b/offload/tools/CMakeLists.txt
@@ -7,7 +7,7 @@ macro(add_offload_tool name)
 endmacro()
 
 macro(add_offload_tool_symlink name)
-  llvm_add_tool_symlink(OFFLOAD ${ARGV})
+  llvm_add_tool_symlink(OFFLOAD ${ARGV} NO_INSTALL_RPATH)
 endmacro()
 
 add_subdirectory(deviceinfo)
